frontend/src/components/TopBar.jsx

```diff
@@ -24,13 +25,14 @@ class TopBar extends React.Component {
           <div className="topBarNavigationItems">
             <ul>
               <li>
-                <NavLink activeClassName="active" to="/">
+                <Modal />
+                {/* <NavLink activeClassName="active" to="/">
```

Comment:
death code ?

Reply:
Dead and buried now, my Cap'tain!

Reply:
Ohnoz, that's undead code, it came back! :scream: